usecases/schema/add.go

```diff
@@ -126,6 +126,17 @@ func (m *Manager) RestoreClass(ctx context.Context, d *backup.ClassDescriptor) e
 	m.triggerSchemaUpdateCallbacks()
```

Comment:
no test?

Reply:
this was only to produce an image for a customer